app/src/test/java/ch/epfl/sdp/peakar/points/ComputePOIPointsUnitTest.java

```diff
@@ -9,6 +10,13 @@
 
 public class ComputePOIPointsUnitTest {
 
+    static ComputePOIPoints computePOIPointsInstance;
+
+    @BeforeClass
+    public static void getInstance() {
+        computePOIPointsInstance = ComputePOIPoints.getInstance(null);
```

Comment:
This produces a NullPointerException. I don't know if it's possible to get the context of the application in the classic unit tests. The easiest solutions would be to extract the tested methods to a utility class or to move the test to the instrumented test package.